src/snyk/snykIac/views/suggestion/iacSuggestionWebviewScript.ts

```diff
@@ -163,7 +163,6 @@
 
   function appendIdentifierSpan(identifiers: Element, id: string, link?: string) {
     const delimiter = document.createElement('span');
-    delimiter.innerText = ' | ';
     delimiter.className = 'delimiter';
     identifiers.appendChild(delimiter);
```

Comment:
When testing https://github.com/dexidp/dex I realised that we had an extra delimiter in **Snyk Configuration Issue** panel

|Before|After|
|-|-|
|<img width="1252" alt="Before" src="https://github.com/snyk/vscode-extension/assets/1948377/c202184c-93cd-4b61-89da-0d177aa743f5">|<img width="1251" alt="After" src="https://github.com/snyk/vscode-extension/assets/1948377/bb4a37a4-6c4a-4241-bf10-9a948a3ef15f">|